.github/workflows/develop.yml

```diff
@@ -140,7 +140,7 @@ jobs:
       - name: Run cargo clippy
         run: cargo clippy --workspace --all-targets -- -D warnings
 
-  coverage:
+  tests:
```

Comment:
```suggestion
  coverage:
```

Reply:
@MichaelScofield Please read https://github.com/GreptimeTeam/greptimedb/pull/2734#issuecomment-1822161656, but we can retain the `coverage` name and change it if better later.